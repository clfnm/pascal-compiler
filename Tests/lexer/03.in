program
_program
Hello
world
WrItE