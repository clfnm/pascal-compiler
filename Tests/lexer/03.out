1	1	Keyword	PROGRAM	program
2	1	Id	_program	_program
3	1	Id	Hello	Hello
4	1	Id	world	world
5	1	Keyword	WRITE	WrItE
5	6	Eof	EOF	